app/parse.py

```diff
@@ -8,5 +15,39 @@ class Course:
     duration: str
 
 
+FIELDS_LIST = [field.name for field in fields(Course)]
+
+
+def parse_single_course(soup: Tag) -> Course:
+    return Course(
+        name=soup.select_one("a.ProfessionCard_title__fEWio > h3").text,
+        short_description=soup.select_one(
+            ".ProfessionCard_professionTags__2iarD + "
+            "p.typography_landingTextMain__Rc8BD"
+        ).text,
+        duration=soup.select_one(
+            "p.ProfessionCardTags_regularTag__dqOGj:last-child span"
+        ).text
+    )
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    text = requests.get(BASE_URL).content
```

Comment:
Consider adding error handling for the HTTP request. If the request fails, the script will raise an exception. You might want to check the response status code and handle exceptions appropriately.